builder/index.html

```diff
@@ -266,11 +264,11 @@ <h3 >{{getMethodName(selectedMethod)}}
                               <form class="form-inline" role="form">
                                 <div class="form-group" ng-show="checkMajorVersion()">
                                   <label for="os_class">Version (Major):</label>
-                                  <input type="text" ng-pattern="versionRegex" style="width:50px" class="form-control" ng-change="updateClassContext()" ng-model="selectedMethod.OS_class.majorVersion"  placeholder=""></input>
+                                  <input type="text" ng-pattern="versionRegex" style="width:50px" class="form-control" ng-change="updateClassContext()" ng-model="selectedMethod.OS_class.majorVersion"  placeholder="">
                                 </div>
                                 <div class="form-group" ng-show="checkMinorVersion()">
                                   <label for="os_class">Version (Minor):</label>
-                                  <input type="text"  style="width:50px" ng-pattern="versionRegex" class="form-control" ng-change="updateClassContext()" ng-disabled="selectedMethod.OS_class.majorVersion === undefined || selectedMethod.OS_class.majorVersion === '' " ng-model="selectedMethod.OS_class.minorVersion"  placeholder=""></input>
```

Comment:
Just in order to clean up the code. Input tag does not have a closing tag if we are referring to the W3C html standard. 
I also removed some other closing tags which were referring to any other tag.